raiden/tests/unit/channel_state/test_coop_settle.py

```diff
@@ -0,0 +1,448 @@
+# pylint: disable=too-many-locals,too-many-statements,too-many-lines
+import random
+
+import pytest
+
+from raiden.tests.unit.channel_state.utils import (
+    assert_partner_state,
+    create_channel_from_models,
+    create_model,
+)
+from raiden.tests.utils.events import search_for_item
+from raiden.tests.utils.factories import make_block_hash
+from raiden.transfer import channel
+from raiden.transfer.events import (
+    ContractSendChannelCoopSettle,
+    EventInvalidReceivedWithdrawRequest,
+    SendProcessed,
+    SendWithdrawConfirmation,
+    SendWithdrawRequest,
+)
+from raiden.transfer.state import (
+    CoopSettleState,
+    PendingWithdrawState,
+    message_identifier_from_prng,
+)
+from raiden.transfer.state_change import (
+    ActionChannelCoopSettle,
+    ReceiveWithdrawConfirmation,
+    ReceiveWithdrawRequest,
+)
+from raiden.utils.packing import pack_withdraw
+from raiden.utils.signer import LocalSigner
+
+
+def make_receive_coop_settle_withdraw_confirmation(
```

Comment:
I don't think there's need to make this public. If it's used just within this one test module, please make it private.

Reply:
Nice, today I learned that `_methods` will not get imported if you do the `from foo import *`
Changed.